scripts/taco_to_burrito.py

```diff
@@ -32,25 +33,30 @@ def main():
             convert_markers(full_path, target_path)
 
 ################################################################################
-# Quick error printing logic that only prints the title of a section if there
-# are any errors in it. Could be handled better by returning errors from the
-# function but that is more work to implement then this is.
+# Quick error printing logic
 ################################################################################
-eprint_title: str = ""
-eprint_trigger: bool = False
-def set_eprint_title(title: str):
-    global eprint_title
-    global eprint_trigger
-    eprint_title = title
-    eprint_trigger = False
-def eprint(*args, **kwargs):
-    global eprint_trigger
-    if not eprint_trigger:
-        eprint_trigger = True
-        print(eprint_title)
-    print("   ", *args, **kwargs)
-
-
+error_location = ""
+class ErrorType(Enum):
+    INFO = 0
+    WARNING = 1
+    ERROR = 2
+    
+def eprint(*args):
+    error_level = ErrorType.WARNING
+    for arg in list(args):
+        if(type(arg) == ErrorType):
+            error_level = arg
+            
+    print(error_level.name + " in file \"" + error_location + "\": ")
+    for arg in list(args):
```

Comment:
This looks like it is going to print each argument on a separate indented line.
```python
eprint("Hello", "World")
```
would print out
```
    Hello
    World
```
when it probably should print out
```
    Hello World
```

This can be fixed if you change the function signature mentioned previously and just call
```python
print("    ", *args, **kwargs)
```
like the previous function did.

Reply:
this is intended, so that you can pass an explanation and the error itself as two different arguments, like this for example:
![image](https://user-images.githubusercontent.com/103769692/213847344-ce8f6dba-d763-40b8-8a5f-45955d1c40fd.png)
the only thing that changes is that you have to do a little bit of string formatting when calling the eprint function which is already being done anyway.

Reply:
I don't see a solid argument for changing the normal functionality and removing all the keyword functionality in favor of mandating a newline per argument. If you want a newline just do the formatting to add the newline instead of require formatting to not add it.